.github/workflows/ci.yaml

```diff
@@ -11,9 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v4
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       - name: Setup Python
-        uses: actions/setup-python@v4
+        uses: actions/setup-python@42375524e23c412d93fb67b49958b491fce71c38 # v5.4.0
```

Comment:
Is it okay to upgrade or should I just pin this to https://github.com/actions/setup-python/commit/b64ffcaf5b410884ad320a9cfac8866006a109aa?

Reply:
@anayeaye @smohiudd